frontend/src/CommitInfo.res

```diff
@@ -40,14 +40,18 @@ let renderExternalLink = (~href, text) => {
   <a target="_blank" className={Sx.make(sx)} href> {Rx.text(text)} </a>
 }
 
+let url: string = %raw(`import.meta.env.VITE_PIPELINE_FRONTEND_URL`)
```

Comment:
I think using `url` is wrong here. URLs include ports and yet port is a separate variable.

A better naming would be: `host` and `port`. See my general comment on naming too.